alphadia/transferlearning/train.py

```diff
@@ -65,10 +48,11 @@ def __init__(self, optimizer: torch.optim.Optimizer, **kwargs):
         self.optimizer = optimizer
         self.num_warmup_steps = kwargs.get("num_warmup_steps", 5)
```

Comment:
just for my understanding: are these kwargs ever being filled anywhere? The only instantiation of the `CustomScheduler` I found was in `peptdeep.model_interface._get_lr_schedule_with_warmup` . Or am I missing something?